tests/drivers/build_all/ethernet/prj.conf

```diff
@@ -7,3 +7,8 @@ CONFIG_TEST_RANDOM_GENERATOR=y
 CONFIG_TEST_USERSPACE=y
 
 CONFIG_GPIO=y
+
+CONFIG_SPI_INIT_PRIORITY=50
+CONFIG_ETH_INIT_PRIORITY=50
+CONFIG_MDIO_INIT_PRIORITY=50
+CONFIG_PHY_INIT_PRIORITY=50
```

Comment:
Builds and works for ADIN with:

```
CONFIG_MDIO_INIT_PRIORITY=70
CONFIG_ETH_INIT_PRIORITY=70
```

Reply:
@fabiobaltieri is there anyway to get it cleaner? Should the driver's Kconfig set MDIO and ETH priorities so that it doesn't require further configuration in the app?

Reply:
Yeah it'd be cool to clean that up, now that we have ordinals in the sequence those priorities do more damage than anything else. I'm working on something more radical to propose soon but in the meantime feel free to change it in the drivers.